submissions/BlueLamantine/friends app/js/main.js

```diff
@@ -0,0 +1,352 @@
+class Render {
+  constructor() {
+    if (Render.instance == null) {
+      this.profilesContainer = document.querySelector('.profile-cards');
+      this.filtersContainer = document.querySelector('#filters');
+      document.querySelector('#menu').addEventListener('click', () => {
+        document.querySelector('#filters').classList.toggle('active');
+      });
+      Render.instance = this;
+    }
+    return Render.instance;
```

Comment:
As I told, YAGNI. In this task can call constructor only once

Reply:
fixed!